.circleci/config.yml

```diff
@@ -455,6 +455,8 @@ jobs:
 
   e2e_tests_adapters:
     <<: *e2e-executor
+    docker:
+      - image: cypress/browsers:node-18.16.1-chrome-114.0.5735.133-1-ff-114.0.2-edge-114.0.1823.51-1
```

Comment:
Newer node than e2e default needed for latest netlify-cli. As gatsby itself has node 18.0.0 min, we should not bump it in other tests so default stayed as is